modules/RTC/TraceablePeerConnection.js

```diff
@@ -1044,6 +1048,47 @@ TraceablePeerConnection.prototype.getLocalSSRC = function(localTrack) {
     return ssrcInfo && ssrcInfo.ssrcs[0];
 };
 
+/**
+ * When doing unified plan simulcast, we'll have a set of ssrcs with the
+ * same msid but no ssrc-group, since unified plan signals the simulcast
+ * group via the a=simulcast line.  Unfortunately, Jicofo will complain
+ * if it sees ssrcs with matching msids but no ssrc-group, so we'll inject
+ * an ssrc-group line to make Jicofo happy.
+ * NOTE: unlike plan B simulcast, the ssrcs in this inject ssrc-group will
+ * NOT necessarily be in order of quality (low to high) because:
+ * a) when translating between unified plan and plan b the order of the ssrcs
+ * is not preserved and
+ * b) it isn't guaranteed that firefox will give them to us in order to begin
+ * with
+ * @param desc A session description object (with 'type' and 'sdp' fields)
+ * @return A session description object with its sdp field modified to
+ * contain an inject ssrc-group for simulcast
+ */
+TraceablePeerConnection.prototype._injectSsrcGroupForUnifiedSimulcast
+    = function(desc) {
+        const sdp = transform.parse(desc.sdp);
+        const video = sdp.media.find(mline => mline.type === 'video');
+
+        if (video.simulcast_03) {
+            const ssrcs = [];
+
+            video.ssrcs.forEach(ssrc => {
+                if (ssrc.attribute === 'msid') {
+                    ssrcs.push(ssrc.id);
+                }
+            });
+            video.ssrcGroups = [
+                {
+                    semantics: 'SIM',
+                    ssrcs: ssrcs.join(' ')
+                }
+            ];
+        }
+        desc.sdp = transform.write(sdp);
+
+        return desc;
```

Comment:
since this is a getter, i think there's a check where we call get to see if we have one already or not